dali/python/backend_impl.cc

```diff
@@ -1288,13 +1290,19 @@ PYBIND11_MODULE(backend_impl, m) {
 
   m.def("GetCxx11AbiFlag", &GetCxx11AbiFlag);
 
-  m.def("HasCudaContext", []{
-    if (!cuInitChecked()) {
-      return false;
+  m.def("IsDriverInitialized", [] {
+    auto this_pid = getpid();
+    std::ifstream maps_file(make_string("/proc/", this_pid, "/maps"));
```

Comment:
```C++
#ifef __unix__```
or something similar?
Also, I don't know how costly this check is - maybe it would be better to try to load?